Trouble with Cameras and Alarm

joey00000 opened the pull request:
Hey, I have just started this homebridge project on a RPi and I am completely new so I apologize in advance for my unfamiliarity.

Everything is working great with my sensors, however my alarm is missing.

I have also been trying to set up the cameras with ffmpeg but I am very confused of how to create a debugging node in terminal. I have been able to get the IP and password but no username.

Any help would be appreciated :)

daymondm commented:
The username for the cameras is just “user”. You only need to use the node terminal to get the ID numbers of the camera.. check: https://github.com/daymondm/homebridge-vivint/issues/1#issuecomment-491478995 

I’ll see if I can be of further help when I’m at home - we did sell our house along with the Vivint system so I’m afraid I may not be able to help debug much.

balansse committed:
Update CHANGELOG.md

balansse committed:
1.2.0

balansse committed:
Add motionTimeout to camera config

Commit:
Merge branch 'develop'

balansse committed:
Move accessories folder under lib

balansse committed:
1.2.1

balansse committed:
Update CHANGELOG.md

balansse committed:
Merge branch 'develop'

balansse committed:
Fix double "this" in thermostat.js

balansse committed:
v1.2.2

balansse committed:
Update CHANGELOG.md

balansse committed:
Add support for new contact sensors
Set device ID as serial number if no serial is provided (HA)

balansse committed:
Append device id to the end of serial number

balansse committed:
Update README.md

balansse committed:
1.3.0

balansse committed:
Fix thermostat handling

balansse committed:
1.3.1

balansse committed:
Update CHANGELOG.md

balansse committed:
Fix Thermostat handling

balansse committed:
1.4.0

balansse committed:
Fix error on old NodeJS versions

balansse committed:
1.4.1

balansse committed:
Fix doorbell button notifications;
Fix panel target value reading for Homebridge 1.3.0

balansse committed:
1.4.2

balansse committed:
More Homebridge 1.3.0 compatibility fixes;
Fix message handling

balansse committed:
Add built-in camera support

balansse committed:
Add logo

balansse committed:
Update homebridge-vivint.svg

balansse committed:
Add files via upload

balansse committed:
Update README.md

balansse committed:
Merge branch 'master' of https://github.com/balansse/homebridge-vivint

balansse committed:
Update package versions
Housekeeping

balansse committed:
Fix lock status handling for Homebridge 1.3.0

balansse committed:
1.4.3

balansse committed:
Merge branch 'master' into ffmpeg

balansse committed:
Add disable cameras settings option

balansse committed:
Change README and CHANGELOG

balansse committed:
Change the way auth login failover works

balansse committed:
1.5.0

balansse committed:
Fix handling cameras with no audio feed
Improve video loading speed
Switch to the direct snapshot retrieval
Remove cameras from the system completely after disableCameras flag was enabled

balansse committed:
1.5.1

balansse committed:
remove logging for cached accessories

balansse committed:
1.5.2

balansse committed:
Add the option to switch video feeds to stream from Vivint servers

balansse committed:
1.5.3

balansse committed:
Use system ffmpeg if local was not found

balansse committed:
1.5.4

arosenb2 committed:
fix: change vivint api host

Several users have been getting errors from Cloudflare due to needing `www` in the API url. This amends the url to contain the correct constant.

https://github.com/balansse/homebridge-vivint/issues/37#issuecomment-913183556

balansse committed:
Merge pull request #40 from arosenb2/patch-1

fix: change vivint api host

jgrimard committed:
Added support for MFA

jgrimard committed:
MFA support

Added persist_session and changed all cookie references to refreshToken.

jgrimard committed:
Temporary changes for test publish to NPM

jgrimard committed:
Possibly fix for "Accessories must be created with a non-empty displayName"

jgrimard committed:
npm rev

jgrimard committed:
Revert "npm rev"

This reverts commit 936f1d6454e06bbe4571981e29badbacc952a34b.

jgrimard committed:
Revert "Possibly fix for "Accessories must be created with a non-empty displayName""

This reverts commit b898166825c924ecb9062fae0a0e58bd24e4b572.

jgrimard committed:
Cleanup and adding build scripts

jgrimard committed:
Cleanup and adding build notes

jgrimard committed:
Cleanup

jgrimard committed:
Changed Rev to 1.6

arjunmehta committed:
Improve publishing process with prepare script and npmignore

arjunmehta committed:
Clean up

arjunmehta committed:
Add arjunmeht to contributors

arjunmehta committed:
Clean up

jgrimard committed:
Update server.js

jgrimard committed:
Merge pull request #1 from arjunmehta/am_improve-package

Improve publishing process with prepare script and npmignore

balansse committed:
Merge pull request #43 from jgrimard/master

Pull request adding MFA support

balansse committed:
Change README and CHANGELOG

balansse committed:
Change README

balansse committed:
1.6.0

jgrimard committed:
Fixing broken links to github page.

I accidentally introduced this error.
This fixes the broken link in to the github page on the homebridge plugins page.
It will also fix broken links on the npmjs page.

jgrimard committed:
Fixed a typo

Thanks for the credit

balansse committed:
Merge pull request #45 from jgrimard/patch-2

Fixed a typo

balansse committed:
Merge pull request #44 from jgrimard/patch-1

Fixing broken Github links

jgrimard committed:
Improving documentation for MFA and HOOBS

balansse committed:
Merge pull request #50 from jgrimard/master

Improving documentation for MFA and HOOBS

balansse committed:
Merge branch 'master' of https://github.com/balansse/homebridge-vivint

mikedecaro committed:
Fix dimmer switch debouncing

balansse committed:
Merge pull request #60 from mikedecaro/master

Fix dimmer switch debouncing

balansse committed:
Update pubnub and ffmpeg packages

balansse committed:
1.6.1

balansse committed:
Update CHANGELOG.md

balansse committed:
Create npm-publish.yml

balansse committed:
Update npm-publish.yml

balansse committed:
Update npm-publish.yml

balansse committed:
Update npm-publish.yml

mikedecaro committed:
Add support for fan rotation speed

balansse committed:
Merge pull request #66 from mikedecaro/master

Add support for fan rotation speed

balansse committed:
Assign default name if no name is received from Vivint

balansse committed:
Add lowBatteryLevel config value to override Vivint low battery level notifications

balansse committed:
Check if BatteryLevel is a number before returning

balansse committed:
Add support for water sensor

balansse committed:
Handle the case when no humidity value is provided

balansse committed:
Add default states for some thermostat values

balansse committed:
Refactor garage door handling;
Fix error logging

balansse committed:
Fix for Garage Door handling

balansse committed:
Update CHANGELOG.md

balansse committed:
1.7.0

balansse committed:
Update CHANGELOG.md

balansse committed:
Update FUNDING.yml

dependabot[bot] committed:
Bump async from 2.6.3 to 2.6.4 in /lib/homebridge-ui

Bumps [async](https://github.com/caolan/async) from 2.6.3 to 2.6.4.
- [Release notes](https://github.com/caolan/async/releases)
- [Changelog](https://github.com/caolan/async/blob/v2.6.4/CHANGELOG.md)
- [Commits](https://github.com/caolan/async/compare/v2.6.3...v2.6.4)

---
updated-dependencies:
- dependency-name: async
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump nanoid from 3.1.25 to 3.3.4 in /lib/homebridge-ui

Bumps [nanoid](https://github.com/ai/nanoid) from 3.1.25 to 3.3.4.
- [Release notes](https://github.com/ai/nanoid/releases)
- [Changelog](https://github.com/ai/nanoid/blob/main/CHANGELOG.md)
- [Commits](https://github.com/ai/nanoid/compare/3.1.25...3.3.4)

---
updated-dependencies:
- dependency-name: nanoid
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump minimist from 1.2.5 to 1.2.6 in /lib/homebridge-ui

Bumps [minimist](https://github.com/substack/minimist) from 1.2.5 to 1.2.6.
- [Release notes](https://github.com/substack/minimist/releases)
- [Commits](https://github.com/substack/minimist/compare/1.2.5...1.2.6)

---
updated-dependencies:
- dependency-name: minimist
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump eventsource from 1.1.0 to 1.1.2 in /lib/homebridge-ui

Bumps [eventsource](https://github.com/EventSource/eventsource) from 1.1.0 to 1.1.2.
- [Release notes](https://github.com/EventSource/eventsource/releases)
- [Changelog](https://github.com/EventSource/eventsource/blob/master/HISTORY.md)
- [Commits](https://github.com/EventSource/eventsource/compare/v1.1.0...v1.1.2)

---
updated-dependencies:
- dependency-name: eventsource
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

balansse committed:
Merge pull request #69 from balansse/dependabot/npm_and_yarn/lib/homebridge-ui/async-2.6.4

Bump async from 2.6.3 to 2.6.4 in /lib/homebridge-ui

balansse committed:
Merge pull request #72 from balansse/dependabot/npm_and_yarn/lib/homebridge-ui/eventsource-1.1.2

Bump eventsource from 1.1.0 to 1.1.2 in /lib/homebridge-ui

dependabot[bot] committed:
Bump url-parse from 1.5.3 to 1.5.10 in /lib/homebridge-ui

Bumps [url-parse](https://github.com/unshiftio/url-parse) from 1.5.3 to 1.5.10.
- [Release notes](https://github.com/unshiftio/url-parse/releases)
- [Commits](https://github.com/unshiftio/url-parse/compare/1.5.3...1.5.10)

---
updated-dependencies:
- dependency-name: url-parse
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

balansse committed:
Merge pull request #73 from balansse/dependabot/npm_and_yarn/lib/homebridge-ui/url-parse-1.5.10

Bump url-parse from 1.5.3 to 1.5.10 in /lib/homebridge-ui

balansse committed:
Merge pull request #71 from balansse/dependabot/npm_and_yarn/lib/homebridge-ui/minimist-1.2.6

Bump minimist from 1.2.5 to 1.2.6 in /lib/homebridge-ui

balansse committed:
Merge pull request #70 from balansse/dependabot/npm_and_yarn/lib/homebridge-ui/nanoid-3.3.4

Bump nanoid from 3.1.25 to 3.3.4 in /lib/homebridge-ui

balansse committed:
Create codeql-analysis.yml

dependabot[bot] committed:
Bump follow-redirects from 1.14.4 to 1.15.1 in /lib/homebridge-ui

Bumps [follow-redirects](https://github.com/follow-redirects/follow-redirects) from 1.14.4 to 1.15.1.
- [Release notes](https://github.com/follow-redirects/follow-redirects/releases)
- [Commits](https://github.com/follow-redirects/follow-redirects/compare/v1.14.4...v1.15.1)

---
updated-dependencies:
- dependency-name: follow-redirects
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

balansse committed:
Merge pull request #74 from balansse/dependabot/npm_and_yarn/lib/homebridge-ui/follow-redirects-1.15.1

Bump follow-redirects from 1.14.4 to 1.15.1 in /lib/homebridge-ui

balansse committed:
Update packages

dependabot[bot] committed:
Bump terser from 5.14.1 to 5.14.2 in /lib/homebridge-ui

Bumps [terser](https://github.com/terser/terser) from 5.14.1 to 5.14.2.
- [Release notes](https://github.com/terser/terser/releases)
- [Changelog](https://github.com/terser/terser/blob/master/CHANGELOG.md)
- [Commits](https://github.com/terser/terser/commits)

---
updated-dependencies:
- dependency-name: terser
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

balansse committed:
Merge pull request #76 from balansse/dependabot/npm_and_yarn/lib/homebridge-ui/terser-5.14.2

Bump terser from 5.14.1 to 5.14.2 in /lib/homebridge-ui

dependabot[bot] committed:
Bump vm2 from 3.9.9 to 3.9.11

Bumps [vm2](https://github.com/patriksimek/vm2) from 3.9.9 to 3.9.11.
- [Release notes](https://github.com/patriksimek/vm2/releases)
- [Changelog](https://github.com/patriksimek/vm2/blob/master/CHANGELOG.md)
- [Commits](https://github.com/patriksimek/vm2/compare/3.9.9...3.9.11)

---
updated-dependencies:
- dependency-name: vm2
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump loader-utils from 2.0.0 to 2.0.4 in /lib/homebridge-ui

Bumps [loader-utils](https://github.com/webpack/loader-utils) from 2.0.0 to 2.0.4.
- [Release notes](https://github.com/webpack/loader-utils/releases)
- [Changelog](https://github.com/webpack/loader-utils/blob/v2.0.4/CHANGELOG.md)
- [Commits](https://github.com/webpack/loader-utils/compare/v2.0.0...v2.0.4)

---
updated-dependencies:
- dependency-name: loader-utils
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

balansse committed:
Merge pull request #84 from balansse/dependabot/npm_and_yarn/vm2-3.9.11

Bump vm2 from 3.9.9 to 3.9.11

balansse committed:
Merge pull request #87 from balansse/dependabot/npm_and_yarn/lib/homebridge-ui/loader-utils-2.0.4

Bump loader-utils from 2.0.0 to 2.0.4 in /lib/homebridge-ui

dependabot[bot] committed:
Bump minimatch and recursive-readdir in /lib/homebridge-ui

Bumps [minimatch](https://github.com/isaacs/minimatch) and [recursive-readdir](https://github.com/jergason/recursive-readdir). These dependencies needed to be updated together.

Updates `minimatch` from 3.0.4 to 3.1.2
- [Release notes](https://github.com/isaacs/minimatch/releases)
- [Commits](https://github.com/isaacs/minimatch/compare/v3.0.4...v3.1.2)

Updates `recursive-readdir` from 2.2.2 to 2.2.3
- [Release notes](https://github.com/jergason/recursive-readdir/releases)
- [Changelog](https://github.com/jergason/recursive-readdir/blob/master/CHANGELOG.md)
- [Commits](https://github.com/jergason/recursive-readdir/commits/v2.2.3)

---
updated-dependencies:
- dependency-name: minimatch
  dependency-type: indirect
- dependency-name: recursive-readdir
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

jgrimard committed:
Fixed typo

balansse committed:
Merge pull request #91 from jgrimard/patch-1

Fixed typo

balansse committed:
Merge pull request #88 from balansse/dependabot/npm_and_yarn/lib/homebridge-ui/minimatch-and-recursive-readdir-3.1.2

Bump minimatch and recursive-readdir in /lib/homebridge-ui

dependabot[bot] committed:
Bump decode-uri-component from 0.2.0 to 0.2.2 in /lib/homebridge-ui

Bumps [decode-uri-component](https://github.com/SamVerschueren/decode-uri-component) from 0.2.0 to 0.2.2.
- [Release notes](https://github.com/SamVerschueren/decode-uri-component/releases)
- [Commits](https://github.com/SamVerschueren/decode-uri-component/compare/v0.2.0...v0.2.2)

---
updated-dependencies:
- dependency-name: decode-uri-component
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump qs from 6.5.2 to 6.5.3

Bumps [qs](https://github.com/ljharb/qs) from 6.5.2 to 6.5.3.
- [Release notes](https://github.com/ljharb/qs/releases)
- [Changelog](https://github.com/ljharb/qs/blob/main/CHANGELOG.md)
- [Commits](https://github.com/ljharb/qs/compare/v6.5.2...v6.5.3)

---
updated-dependencies:
- dependency-name: qs
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

balansse committed:
Merge pull request #97 from balansse/dependabot/npm_and_yarn/qs-6.5.3

Bump qs from 6.5.2 to 6.5.3

balansse committed:
Merge pull request #95 from balansse/dependabot/npm_and_yarn/lib/homebridge-ui/decode-uri-component-0.2.2

Bump decode-uri-component from 0.2.0 to 0.2.2 in /lib/homebridge-ui

LuisRodriguezLD committed:
add phillips_hue_bridge_device to the ignore list

dependabot[bot] committed:
Bump json5 from 1.0.1 to 1.0.2 in /lib/homebridge-ui

Bumps [json5](https://github.com/json5/json5) from 1.0.1 to 1.0.2.
- [Release notes](https://github.com/json5/json5/releases)
- [Changelog](https://github.com/json5/json5/blob/main/CHANGELOG.md)
- [Commits](https://github.com/json5/json5/compare/v1.0.1...v1.0.2)

---
updated-dependencies:
- dependency-name: json5
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

LuisRodriguezLD committed:
add multilevel_switch_device to the ignore list

balansse committed:
Merge pull request #101 from balansse/dependabot/npm_and_yarn/lib/homebridge-ui/json5-1.0.2

Bump json5 from 1.0.1 to 1.0.2 in /lib/homebridge-ui

dependabot[bot] committed:
Bump cookiejar from 2.1.3 to 2.1.4

Bumps [cookiejar](https://github.com/bmeck/node-cookiejar) from 2.1.3 to 2.1.4.
- [Release notes](https://github.com/bmeck/node-cookiejar/releases)
- [Commits](https://github.com/bmeck/node-cookiejar/commits)

---
updated-dependencies:
- dependency-name: cookiejar
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

balansse committed:
Merge pull request #102 from balansse/dependabot/npm_and_yarn/cookiejar-2.1.4

Bump cookiejar from 2.1.3 to 2.1.4

dependabot[bot] committed:
Bump webpack from 5.73.0 to 5.76.1 in /lib/homebridge-ui

Bumps [webpack](https://github.com/webpack/webpack) from 5.73.0 to 5.76.1.
- [Release notes](https://github.com/webpack/webpack/releases)
- [Commits](https://github.com/webpack/webpack/compare/v5.73.0...v5.76.1)

---
updated-dependencies:
- dependency-name: webpack
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump vm2 from 3.9.11 to 3.9.19

Bumps [vm2](https://github.com/patriksimek/vm2) from 3.9.11 to 3.9.19.
- [Release notes](https://github.com/patriksimek/vm2/releases)
- [Changelog](https://github.com/patriksimek/vm2/blob/master/CHANGELOG.md)
- [Commits](https://github.com/patriksimek/vm2/compare/3.9.11...3.9.19)

---
updated-dependencies:
- dependency-name: vm2
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

balansse committed:
Add support for new wireless_sensor device types;
Add possibility to ignore devices by equipment code and id;
Add config option to log all configured device information;

balansse committed:
Expose doorbell button as contact sensor;
Add debounce to doorbell notifications to prevent double notifications;

balansse committed:
Remove debounce from doorbell press handler

balansse committed:
Add showCameraConfig back

balansse committed:
Add new doorbell button sensor without readding the doorbell accessory

balansse committed:
Merge branch 'feature/showCameraConfig' into release/1.8.0

balansse committed:
Merge branch 'feature/doorbell-button-extract' into release/1.8.0

balansse committed:
Update CHANGELOG.md

balansse committed:
1.8.0

balansse committed:
Merge pull request #104 from balansse/dependabot/npm_and_yarn/lib/homebridge-ui/webpack-5.76.1

Bump webpack from 5.73.0 to 5.76.1 in /lib/homebridge-ui

balansse committed:
Merge pull request #112 from balansse/dependabot/npm_and_yarn/vm2-3.9.19

Bump vm2 from 3.9.11 to 3.9.19

balansse committed:
Merge pull request #113 from balansse/release/1.8.0

Release/1.8.0

balansse committed:
Update readme

balansse committed:
1.8.1

LuisRodriguezLD committed:
Merge branch 'master' into master

balansse committed:
Merge pull request #100 from LuisRodriguezLD/master

Add devices to the ignore list

balansse committed:
Merge branch 'master' of https://github.com/balansse/homebridge-vivint

balansse committed:
Add UI manual entry to device ignore list

balansse committed:
Update CHANGELOG.md

balansse committed:
Fix version number

balansse committed:
1.8.1

balansse committed:
minor bugfix

balansse committed:
1.8.2

balansse committed:
Convert ignorelist IDs to string

balansse committed:
1.8.3

dependabot[bot] committed:
Bump tough-cookie from 4.0.0 to 4.1.3 in /lib/homebridge-ui

Bumps [tough-cookie](https://github.com/salesforce/tough-cookie) from 4.0.0 to 4.1.3.
- [Release notes](https://github.com/salesforce/tough-cookie/releases)
- [Changelog](https://github.com/salesforce/tough-cookie/blob/master/CHANGELOG.md)
- [Commits](https://github.com/salesforce/tough-cookie/compare/v4.0.0...v4.1.3)

---
updated-dependencies:
- dependency-name: tough-cookie
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump semver from 7.3.7 to 7.5.4

Bumps [semver](https://github.com/npm/node-semver) from 7.3.7 to 7.5.4.
- [Release notes](https://github.com/npm/node-semver/releases)
- [Changelog](https://github.com/npm/node-semver/blob/main/CHANGELOG.md)
- [Commits](https://github.com/npm/node-semver/compare/v7.3.7...v7.5.4)

---
updated-dependencies:
- dependency-name: semver
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump word-wrap from 1.2.3 to 1.2.4 in /lib/homebridge-ui

Bumps [word-wrap](https://github.com/jonschlinkert/word-wrap) from 1.2.3 to 1.2.4.
- [Release notes](https://github.com/jonschlinkert/word-wrap/releases)
- [Commits](https://github.com/jonschlinkert/word-wrap/compare/1.2.3...1.2.4)

---
updated-dependencies:
- dependency-name: word-wrap
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump word-wrap from 1.2.3 to 1.2.4

Bumps [word-wrap](https://github.com/jonschlinkert/word-wrap) from 1.2.3 to 1.2.4.
- [Release notes](https://github.com/jonschlinkert/word-wrap/releases)
- [Commits](https://github.com/jonschlinkert/word-wrap/compare/1.2.3...1.2.4)

---
updated-dependencies:
- dependency-name: word-wrap
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

balansse committed:
Merge pull request #120 from balansse/dependabot/npm_and_yarn/word-wrap-1.2.4

Bump word-wrap from 1.2.3 to 1.2.4

balansse committed:
Merge pull request #118 from balansse/dependabot/npm_and_yarn/semver-7.5.4

Bump semver from 7.3.7 to 7.5.4

balansse committed:
Merge pull request #117 from balansse/dependabot/npm_and_yarn/lib/homebridge-ui/tough-cookie-4.1.3

Bump tough-cookie from 4.0.0 to 4.1.3 in /lib/homebridge-ui

balansse committed:
Merge pull request #119 from balansse/dependabot/npm_and_yarn/lib/homebridge-ui/word-wrap-1.2.4

Bump word-wrap from 1.2.3 to 1.2.4 in /lib/homebridge-ui

dependabot[bot] committed:
Bump @babel/traverse from 7.18.8 to 7.23.2 in /lib/homebridge-ui

Bumps [@babel/traverse](https://github.com/babel/babel/tree/HEAD/packages/babel-traverse) from 7.18.8 to 7.23.2.
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.23.2/packages/babel-traverse)

---
updated-dependencies:
- dependency-name: "@babel/traverse"
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

jgrimard committed:
Fixing due to Vivint no longer sending set-cookie header from /authuser

jgrimard committed:
Fixing due to Vivint no longer sending set-cookie header from /authuser

balansse committed:
Merge pull request #124 from balansse/dependabot/npm_and_yarn/lib/homebridge-ui/babel/traverse-7.23.2

balansse committed:
Merge pull request #127 from jgrimard/patch-2

balansse committed:
Update package.json

zackman0010 committed:
Handle session cookies by adding a filter on cookie name

zackman0010 committed:
Copy/paste error

zackman0010 committed:
Missed a refresh token

balansse committed:
Merge pull request #128 from zackman0010/patch-1

balansse committed:
Update package.json

dependabot[bot] committed:
Bump follow-redirects from 1.15.1 to 1.15.4 in /lib/homebridge-ui

Bumps [follow-redirects](https://github.com/follow-redirects/follow-redirects) from 1.15.1 to 1.15.4.
- [Release notes](https://github.com/follow-redirects/follow-redirects/releases)
- [Commits](https://github.com/follow-redirects/follow-redirects/compare/v1.15.1...v1.15.4)

---
updated-dependencies:
- dependency-name: follow-redirects
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump ip from 1.1.5 to 1.1.9

Bumps [ip](https://github.com/indutny/node-ip) from 1.1.5 to 1.1.9.
- [Commits](https://github.com/indutny/node-ip/compare/v1.1.5...v1.1.9)

---
updated-dependencies:
- dependency-name: ip
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

balansse committed:
Merge pull request #136 from balansse/dependabot/npm_and_yarn/ip-1.1.9

Bump ip from 1.1.5 to 1.1.9

balansse committed:
Merge pull request #135 from balansse/dependabot/npm_and_yarn/lib/homebridge-ui/follow-redirects-1.15.4

Bump follow-redirects from 1.15.1 to 1.15.4 in /lib/homebridge-ui

dependabot[bot] committed:
Bump follow-redirects from 1.15.4 to 1.15.6 in /lib/homebridge-ui

Bumps [follow-redirects](https://github.com/follow-redirects/follow-redirects) from 1.15.4 to 1.15.6.
- [Release notes](https://github.com/follow-redirects/follow-redirects/releases)
- [Commits](https://github.com/follow-redirects/follow-redirects/compare/v1.15.4...v1.15.6)

---
updated-dependencies:
- dependency-name: follow-redirects
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump webpack-dev-middleware from 5.3.3 to 5.3.4 in /lib/homebridge-ui

Bumps [webpack-dev-middleware](https://github.com/webpack/webpack-dev-middleware) from 5.3.3 to 5.3.4.
- [Release notes](https://github.com/webpack/webpack-dev-middleware/releases)
- [Changelog](https://github.com/webpack/webpack-dev-middleware/blob/v5.3.4/CHANGELOG.md)
- [Commits](https://github.com/webpack/webpack-dev-middleware/compare/v5.3.3...v5.3.4)

---
updated-dependencies:
- dependency-name: webpack-dev-middleware
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump express from 4.18.1 to 4.19.2 in /lib/homebridge-ui

Bumps [express](https://github.com/expressjs/express) from 4.18.1 to 4.19.2.
- [Release notes](https://github.com/expressjs/express/releases)
- [Changelog](https://github.com/expressjs/express/blob/master/History.md)
- [Commits](https://github.com/expressjs/express/compare/4.18.1...4.19.2)

---
updated-dependencies:
- dependency-name: express
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump tar from 6.1.11 to 6.2.1

Bumps [tar](https://github.com/isaacs/node-tar) from 6.1.11 to 6.2.1.
- [Release notes](https://github.com/isaacs/node-tar/releases)
- [Changelog](https://github.com/isaacs/node-tar/blob/main/CHANGELOG.md)
- [Commits](https://github.com/isaacs/node-tar/compare/v6.1.11...v6.2.1)

---
updated-dependencies:
- dependency-name: tar
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump ejs from 3.1.8 to 3.1.10 in /lib/homebridge-ui

Bumps [ejs](https://github.com/mde/ejs) from 3.1.8 to 3.1.10.
- [Release notes](https://github.com/mde/ejs/releases)
- [Commits](https://github.com/mde/ejs/compare/v3.1.8...v3.1.10)

---
updated-dependencies:
- dependency-name: ejs
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

erceth committed:
added lighting groups to accessories

balansse committed:
Merge pull request #138 from balansse/dependabot/npm_and_yarn/lib/homebridge-ui/follow-redirects-1.15.6

Bump follow-redirects from 1.15.4 to 1.15.6 in /lib/homebridge-ui

balansse committed:
Merge pull request #139 from balansse/dependabot/npm_and_yarn/lib/homebridge-ui/webpack-dev-middleware-5.3.4

Bump webpack-dev-middleware from 5.3.3 to 5.3.4 in /lib/homebridge-ui

balansse committed:
Merge pull request #140 from balansse/dependabot/npm_and_yarn/lib/homebridge-ui/express-4.19.2

Bump express from 4.18.1 to 4.19.2 in /lib/homebridge-ui

balansse committed:
Merge pull request #141 from balansse/dependabot/npm_and_yarn/tar-6.2.1

Bump tar from 6.1.11 to 6.2.1

dependabot[bot] committed:
Bump braces from 3.0.2 to 3.0.3

Bumps [braces](https://github.com/micromatch/braces) from 3.0.2 to 3.0.3.
- [Changelog](https://github.com/micromatch/braces/blob/master/CHANGELOG.md)
- [Commits](https://github.com/micromatch/braces/compare/3.0.2...3.0.3)

---
updated-dependencies:
- dependency-name: braces
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

balansse committed:
Merge pull request #145 from balansse/dependabot/npm_and_yarn/lib/homebridge-ui/ejs-3.1.10

Bump ejs from 3.1.8 to 3.1.10 in /lib/homebridge-ui

balansse committed:
Merge pull request #147 from balansse/dependabot/npm_and_yarn/braces-3.0.3

Bump braces from 3.0.2 to 3.0.3

balansse committed:
Merge pull request #146 from erceth/lighting-groups

Add Lighting Groups to Accessories

balansse committed:
Update package.json

Commit:
launch

Commit:
Homebridge 2.0 changes

Commit:
Homebridge 2.0 Changes

jgrimard committed:
Delete .vscode directory

Commit:
Homebridge 2.0 changes

jgrimard committed:
Homebridge V2

Commit:
fixed package.json

mbentley committed:
Added garage tilt sensor; fixes #125

Signed-off-by: Matt Bentley <mbentley@mbentley.net>

balansse committed:
Merge pull request #165 from mbentley/new-sensor-support

Added garage tilt sensor; fixes #125

balansse committed:
Update package.json

balansse committed:
Merge pull request #164 from jgrimard/master

Homebridge 2.0 support

meek2100 committed:
Add additional types to irrelevantDeviceTypes.

control4_device - Integration with Control4
lgit_poe_wifi_bridge_device - Wifi Bridge for Outdoor Camera
mqtt_audio_sync_service - ? Some internal service
holiday_theme_service' - ? Likely for the holiday chimes on doorbells/panels.

meek2100 committed:
Create sanitize_name.js, so sanitizeDeviceName can be reused.

meek2100 committed:
sanitizeDeviceName in device_set.js

meek2100 committed:
Forgot to import the Sanitize function.

meek2100 committed:
Sanitize camera device names

meek2100 committed:
Sanitize device names in carbon_monoxide_sensor.js

meek2100 committed:
Remove unneeded semi-colons.

meek2100 committed:
Sanitize device names in contact_sensor.js

meek2100 committed:
Sanitize device names in dimmer_switch.js

meek2100 committed:
Sanitize device names in garage_door.js

meek2100 committed:
Sanitize device names in light_group.js

meek2100 committed:
Sanitize device names in light_switch.js

meek2100 committed:
Sanitize device names in lock.js

meek2100 committed:
Sanitize device names in motion_sensor.js

meek2100 committed:
Sanitize device names in panel.js

meek2100 committed:
Sanitize device names in smoke_sensor.js

meek2100 committed:
Sanitize device names in thermostat.js

meek2100 committed:
Sanitize device names in device.js

meek2100 committed:
Clean up hidden characters and spacing.

meek2100 committed:
Update sanitize_name.js

meek2100 committed:
Update comment on sanitize_name.js

meek2100 committed:
Clean up some spacing and hidden characters.

meek2100 committed:
Clean up some spacing and hidden characters.

meek2100 committed:
Update package.json

meek2100 committed:
Update CHANGELOG.md

meek2100 committed:
Set a default name for plugin to resolve child-bridge plugin name conflicts.

This default name resolves the issue where you end up with invalid character warnings due to the plugin's name when converting to a child bridge since symbols like "@" or "-" are not supported naming characters in HomeKit.

balansse committed:
Merge pull request #167 from meek2100/patch-1

Add additional types to irrelevantDeviceTypes.

balansse committed:
Merge pull request #169 from meek2100/patch-2

Sanative Device Names to prevent incompatibility or HomeKit errors.

balansse committed:
Update package.json

meek2100 committed:
Merge branch 'balansse:master' into patch-3

meek2100 committed:
Update CHANGELOG.md

balansse committed:
Merge pull request #170 from meek2100/patch-3

Update CHANGELOG.md

GoodOlClint committed:
Fix Vivint camera audio streaming

- Remove -f null from audio args (was discarding audio output)
- Simplify RTSP source args to use reliable tcp transport
- Fix video format conflicts (remove conflicting -f rawvideo)
- Use libx264 for reliable HomeKit video encoding
- Keep working libfdk_aac + aac_eld audio configuration
- Maintain single-process ffmpeg approach for stability

Result: Both video and audio now working in HomeKit

balansse committed:
Merge pull request #175 from GoodOlClint/master

Fix Vivint camera audio streaming

balansse committed:
Update package.json

jgrimard committed:
Update vivint_mfa_cli.js

jgrimard committed:
Removed filter to fix Authentication Issue #179

jgrimard committed:
Removed filter to fix Authentication Issue #179

balansse committed:
Merge pull request #180 from jgrimard/master

Removed .filter to fix Authentication Issue #179

balansse committed:
Update package.json
